scopes/scope/scope/ui/components-drawer/components.drawer.tsx

```diff
@@ -51,7 +50,7 @@ export class ComponentsDrawer implements DrawerType {
 
     if (!scope) return <FullLoader />;
     if (scope.components.length === 0)
-      return <Text className={classNames(mutedItalic, ellipsis, styles.emptyScope)}>Scope is empty</Text>;
+      return <span className={classNames(mutedItalic, ellipsis, styles.emptyScope)}>Scope is empty</span>;
```

Comment:
why? it should be able to use, no?
or is this the same as icon button?

Reply:
`<Text/>` is deprecated and using symphony env. destroy!